source/raphael.core.js

```diff
@@ -5330,6 +5341,8 @@
         next,
         prev,
         timestamp,
+        tempDiff,
```

Comment:
It's still looking a bit out of place. Can be more graceful. To me it looks like it's the job a a separate function to return an object of changed attributes. Anyway, can ne optimised later.